Frameworks/OakFileBrowser/src/FSOutlineViewDelegate.mm

```diff
@@ -219,17 +277,17 @@ - (void)updateConstraints
 	[super updateConstraints];
 }
 
-- (NSInteger)labelIndex
+- (NSArray<OakFinderTag*>*)finderTags
 {
-	return _labelSwatchView ? _labelSwatchView.labelIndex : 0;
+	return _labelSwatchView ? _labelSwatchView.finderTags : @[ ];
 }
 
-- (void)setLabelIndex:(NSInteger)newLabelIndex
+- (void)setFinderTags:(NSArray<OakFinderTag*>*)newFinderTags
 {
-	if(self.labelIndex == newLabelIndex)
+	if([self.finderTags isEqual:newFinderTags])
 		return;
 
-	if(newLabelIndex == 0)
+	if(!newFinderTags && [newFinderTags count] == 0)
```

Comment:
This should probably be `if(!newFinderTags || [newFinderTags count] == 0)` or possibly just `if(![newFinderTags count])` (relying on sending `count` to `nil` returning `nil`).

Reply:
Thanks for catching this!